config/locales/en.yml

```diff
@@ -247,6 +247,26 @@ en:
       para1: "The following abuse report has been sent to the Abuse team:"
       url_para: "URL of the page the abuse is on:"
       description_para: "Description of the abuse:"
+    challenge_assignment_notification:
+      subject: "[%{app_name}] [%{collection_title}] Your Assignment!"
+      recipient: "Recipient:" # this is followed by the name of the gift recipient
+      recipient_missing: "None: contact a moderator for help!"
+      prompts: "Prompts:"
+      optional_tags: "Optional Tags:"
+      prompt_url: "Prompt URL:"
+      description: "Description:"
+      due: "This assignment is due at:"
+      any: "Any"
+      text:
+        part1: "You have been assigned the following request in the %{collection_title} (%{collection_url}) challenge at the Archive of Our Own!"
+        look_up: "You can look up this assignment from your Assignments page at %{link}."
+        footer: "You're receiving this email because you signed up for the %{title} challenge (%{url}). For more information about this challenge and contact information for the moderators, please see %{profile_url}."
```

Comment:
We can make a separate issue for this if you prefer, but I just noticed the text email says "please see" while the HTML one says "please visit," which is probably better phrasing.